spec/features/show_has_correct_content_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../spec_helper'
+require_relative 'load_database'
+
+include LoadDatabase
+
+describe 'user goes to show page' do
+  it 'sees page title' do
+    LoadDatabase.load
+    visit('/stations/1')
+
+    within('h1') {expect(page).to have_content('Station Information')}
+  end
+
+  it 'sees station information' do
+    LoadDatabase.load
+    stations = Station.all.count
```

Comment:
Here, we could call Station.all.each and get rid of a couple of lines. Right now, if we searched for an id that didn't exist (say it had been deleted), it might break this test.